### PR TITLE
complete es6-destructuring

### PR DESCRIPTION
**Pull request by t-doan**
es6-destructuring

### CODE REVIEW

File: es6-destructuring/main.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    author: 'Inio Asano',
+    libraryID: 1233,
+  },
+];
+
+function getBooks() {
+  return library;
+}
+
+const [book3, book4, book5] = getBooks();
```
Comment (shawnkost) [inline es6-destructuring/main.js:50]: No function. They just did this in the example. You just need to to `= library`

File: es6-destructuring/main.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+console.log('book3:', book3);
+console.log('book4:', book4);
+console.log('book5:', book5);
+
+const [, , , book6] = getBooks();
```
Comment (shawnkost) [inline es6-destructuring/main.js:56]: No function. They just did this in the example. You just need to to `= library`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Tai!

File: es6-destructuring/main.js [hunk 3]
```diff
@@ -43,16 +43,12 @@ const library = [
...
-
-const [book3, book4, book5] = getBooks();
+const [book3, book4, book5] = library;
 
 console.log('book3:', book3);
 console.log('book4:', book4);
 console.log('book5:', book5);
 
-const [, , , book6] = getBooks();
+const [, , , book6] = library;
```
Comment (t-doan) [inline es6-destructuring/main.js:52]: Removed the function and assigned to library array.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**